app/src/main/java/com/android/shelfLife/ui/profile/ProfileScreen.kt

```diff
@@ -70,6 +70,23 @@ fun ProfileScreen(
 
   val thresholdTime = 500 // Time in milliseconds between successive clicks
 
+  fun easterEggOnClickCount() {
```

Comment:
I really like this!!! not only did you reduce code duplication but now if we want something else with a similar "click 5 times to trigger" functionality, we can easily reuse it!!!